packages/integration-tests/src/entities.ts

```diff
@@ -0,0 +1,77 @@
+export enum Buffer {
```

Comment:
What is Buffer? Can we use some dynamic path instead based on contract name, test name or whatever it is related to?

Reply:
As far as I know this is the initial way of implementation. I have verified, such changes could be possible but requires a lot of (time) to change almost all dependent files. We could prioritize it for later as a separate task or keep it as is (if it is not an issue). what do you think? @Romsters @abilevych